src/ftvhelp.cpp

```diff
@@ -491,11 +491,11 @@ void FTVHelp::Private::generateTree(TextStream &t, const FTVNodes &nl,int level,
       {
         t << "<span class=\"icondoc\"></span>";
       }
+      generateLink(t,n);
       if (srcRef)
       {
         t << "</a>";
```

Comment:
This closes the link for the icon (pointing to the source file, not the documentation)! The text should point to the documentation (if available), so moving `generateLink` up will produce an invalid nested link.

Reply:
Indeed. I did not test enough. Sorry.

I thought that this would work:
`<a href="https://web.de">weeeeeeb<a href="https://google.com">goooogle .com</a>.de</a>`